src/parse.ts

```diff
@@ -239,6 +247,22 @@ function getPositionFallbackRules(node: csstree.CssNode) {
   return {};
 }
 
+function getCSSPropertyValue(el: HTMLElement, prop: string) {
+  return getComputedStyle(el).getPropertyValue(prop);
```

Comment:
```suggestion
  return getComputedStyle(el).getPropertyValue(prop).trim();
```

It seems like different browsers serialize spaces in custom properties in different ways. Here any spaces at the start and at the end of the value shouldn't have any effect, so we trim them.